### PR TITLE
Run update_deps; migrate to use crane

### PR DESCRIPTION
**Pull request by jacob-delgado**
**Please provide a description of this PR:**
Run bin/update_deps.sh
Use crane now that it is part of the build image

### CODE REVIEW

**Comment by jacob-delgado** [comment]
cc @howardjohn Looks like https://github.com/istio/pkg/pull/438 got rid of the `probe` module and https://github.com/istio/istio/blob/0d240a70072f5e365f4fc24194db1fdfb87dadb1/security/pkg/pki/ca/ca.go#L34 depends on it.

**Comment by howardjohn** [comment]
> cc @howardjohn Looks like [istio/pkg#438](https://github.com/istio/pkg/pull/438) got rid of the `probe` module and
> 
> https://github.com/istio/istio/blob/0d240a70072f5e365f4fc24194db1fdfb87dadb1/security/pkg/pki/ca/ca.go#L34
> 
> depends on it.

It was intended, the dependency is from dead code. I thought I merge a removal already, sorry. Want to fix it here, or I can in another PR?

Basically remove all usage of `probe`, its just a struct field but not used.

Sorry, forgot about this.

**Comment by jacob-delgado** [comment]
> 

I can do it here. Just wanted to make sure it was intended.

File: bin/update_deps.sh [hunk 1]
```diff
@@ -47,8 +47,8 @@ go mod tidy
 sed -i "s/^BUILDER_SHA=.*\$/BUILDER_SHA=$(getSha release-builder)/" prow/release-commit.sh
 sed -i '/PROXY_REPO_SHA/,/lastStableSHA/ { s/"lastStableSHA":.*/"lastStableSHA": "'"$(getSha proxy)"'"/  }' istio.deps
 
-LATEST_DEB11_DISTROLESS_SHA256=$(docker image pull gcr.io/distroless/static-debian11 | grep Digest | awk -F\\: '{print $3}')
+LATEST_DEB11_DISTROLESS_SHA256=$(crane digest gcr.io/distroless/static-debian11 | grep Digest | awk -F\\: '{print $3}')
```
Comment (howardjohn) [inline bin/update_deps.sh:50]: I think the grep part is now wrong?

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
hold for the crane issue

File: bin/update_deps.sh (hunk 1 above)
Comment (ericvn) [reply bin/update_deps.sh:50]: Agree that on my Mac the `Digest` is now missing.

crane digest gcr.io/distroless/static-debian11 | awk -F\\: '{print $2}' seems to be correct.

File: bin/update_deps.sh [hunk 2]
```diff
@@ -47,8 +47,8 @@ go mod tidy
...
 sed -i -E "s/sha256:[a-z0-9]+/sha256:${LATEST_DEB11_DISTROLESS_SHA256}/g" docker/Dockerfile.distroless
 
-LATEST_IPTABLES_DISTROLESS_SHA256=$(docker image pull gcr.io/istio-release/iptables | grep Digest | awk -F\\: '{print $3}')
+LATEST_IPTABLES_DISTROLESS_SHA256=$(crane digest gcr.io/istio-release/iptables | grep Digest | awk -F\\: '{print $3}')
```
Comment (ericvn) [inline bin/update_deps.sh:53]: Same as above.

File: bin/update_deps.sh (hunk 1 above)
Comment (jacob-delgado) [reply bin/update_deps.sh]: Removed the grep and updated the awk command to get rid of the warning.

File: bin/update_deps.sh (hunk 2 above)
Comment (jacob-delgado) [reply bin/update_deps.sh]: Fixed

**Pull request merged**